argoverse/data_loading/simple_track_dataloader.py

```diff
@@ -111,7 +111,7 @@ def get_closest_lidar_fpath(self, log_id: str, cam_timestamp: int) -> Optional[s
         if lidar_timestamp is None:
             return None
         lidar_dir = f"{self.data_dir}/{log_id}/lidar"
-        ply_fname = f"PC_{lidar_timestamp}.ply"
+        ply_fname = f"PC_{lidar_timestamp}.pt"
```

Comment:
note: I will move this to a new dataloader, `class TbvDataloader(SimpleArgoverseTrackingDataLoader)`